tests/test_isrTask.py

```diff
@@ -404,7 +404,7 @@ def test_failCases(self):
         with self.assertRaises(RuntimeError):
             self.validateIsrResults()
 
-    def test_maskingCase_noMasking(self):
+    def test_maskingCase_negativeVariance(self):
```

Comment:
Why don't you just want to add a new test instead of altering this one?

Reply:
That was my original plan, but it looks like I never pasted a copy of the original test before editing.  I've restored the missing test.